apps/frontpage/components/home/social-validation/social-validation.tsx

```diff
@@ -313,46 +316,46 @@ export function SocialValidation({
         </div>
         <div className={cn(socialCard)}>
           <div className="flex h-12 w-12 items-center justify-center">
-            <svg
-              aria-label="Twitter"
-              fill="none"
-              height="40"
-              viewBox="0 0 14 14"
-              width="40"
-              xmlns="http://www.w3.org/2000/svg"
-            >
-              <path
-                d="M11.02.446h2.137L8.49 5.816l5.51 7.28H9.67L6.298 8.683l-3.88 4.413H.282l5.004-5.735L0 .446h4.442l3.064 4.048L11.02.446zm-.759 11.357h1.18L3.796 1.655H2.502l7.759 10.148z"
-                fill="#fff"
-              />
-            </svg>
+            <Youtube />
           </div>
           <div className="text-md text-white">
-            Get the latest news and updates from Storybook maintainers.
+            Watch tutorials, feature previews, and interviews.
           </div>
           <Button asChild jumpOnHover rounded="full" variant="outlineHome">
-            <a href="https://twitter.com/storybookjs">Follow on X</a>
+            <a href="https://www.youtube.com/channel/UCr7Quur3eIyA_oe8FNYexfg">
+              Watch on YouTube
+            </a>
           </Button>
           <div>
-            <div className="text-sm text-white">24,200+</div>
-            <div className="text-sm text-zinc-500">Followers</div>
+            <div className="text-sm text-white">8,320+</div>
+            <div className="text-sm text-zinc-500">Subscribers</div>
           </div>
         </div>
         <div className={cn(socialCard)}>
-          <div className="flex h-12 w-12 items-center justify-center">
-            <Youtube />
+          <div className="flex items-center gap-4">
+            <div className="flex h-12 w-12 items-center justify-center">
+              <Image src={BlueskySvg} alt="BlueSky" className="h-full w-full" />
+            </div>
+            <div className="flex h-12 w-12 items-center justify-center rounded-md outline outline-[rgba(255,255,255,0.1)]">
```

Comment:
`outline outline-[rgba(255,255,255,0.1)]`

Why was this included? It's the only logo with an outline in this section, so it looks out of place to me.

![](https://github.com/user-attachments/assets/f30cc11d-bec8-4172-af0c-5c430d84e8b7)

Reply:
the black on black felt a bit weird. But I'm okay to take it out.

Reply:
![CleanShot 2025-05-20 at 12 24 49@2x](https://github.com/user-attachments/assets/f79f4cf1-9234-4fe6-a26b-c88312ba9d31)

Reply:
Yeah, I see what you're saying, but I think no outline is better. Thanks.